inst/extdata/gcam-china/gdp_mil10usd_province.csv

```diff
@@ -0,0 +1,37 @@
+# File: gdp_mil10usd_province.csv
+# Title: GDP by province
+# Units: million 2010 USD
+# Description: Provincial GDP by historical and future years
+# Source: Citation or source if available
```

Comment:
data source needed @sypnl

Reply:
@rynacui Historical data (1975-2013) on GDP per capita were obtained from the 2014 China Statistics Yearbook and the China Compendium of 60-year Statistics (1949-2008) as in current CNY. The numbers were then all converted to 2010CNY using the World Bank GDP deflators for China (World Development Indicators Database, 2014). All the numbers were converted to 2010USD using the annual average USD-CNY exchange rate in 2010. IMF projected growth rates were applied to project GDP per capita from 2014 to 2020 assuming the same growth rate in 2019 for 2020.  Growth rates from 2021 to 2100 came from the OECD SSP2 Database (IIASA SSP Database, 2015).

Reply:
This input file is total GDP. Are total GDP calculated from pcap GDP and population?

Reply:
yep.